### DISCUSSION TITLE
Can I set visidata to auto reload the file?

### DISCUSSION THREAD

**Discussion by ariel-frischer** [saulpw/visidata#1461] [Q&A]
Is this doable? Whenever the file changes externally or I guess on a timer?

**Comment by frosencrantz** [comment]
Take a look at this thread: https://github.com/saulpw/visidata/issues/683

**Reply by saulpw** [reply]
Yup!  The `reload-every` command is also in vdplus.

**Comment by AllergicMushroom** [comment]
This solution works good, but demands that we call watch-sheet on startup. Is there a way to enable it by default in the .visidatarc ?

**Reply by midichef** [reply]
You can record a series of commands to be carried out on startup. Do the commands you want manually, then press `Ctrl+D` (`save-cmdlog`) and save them to a `.vdj` command logfile.

Quit visidata and rerun it with `-p` to replay the command log: `vd data.tsv -p reloader_cmdlog.vdj`
